automated install/basic-install.sh

```diff
@@ -1240,8 +1238,6 @@ appendToListsFile() {
     case $1 in
         StevenBlack  )  echo "https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts" >> "${adlistFile}";;
         MalwareDom   )  echo "https://mirror1.malwaredomains.com/files/justdomains" >> "${adlistFile}";;
-        DisconTrack  )  echo "https://s3.amazonaws.com/lists.disconnect.me/simple_tracking.txt" >> "${adlistFile}";;
```

Comment:
Are both lists deprecated? Just wondering :)

Reply:
Per the response from disconnect.me to the op of #3450 

>The list you referenced was used in our legacy products. It is not maintained, has not been updated, and is not actively distributed us. Airbrake is not included on our main tracker list (linked below) which is used by partners such as Firefox.

>https://github.com/disconnectme/disconnect-tracking-protection/blob/master/services.json

I'm making the reasonable assumption that both lists are no longer updated/distributed. Besides which, when removed only 337 domains are "lost" from an install that has all lists selected.